test/controllers/quotes_controller_test.rb

```diff
@@ -1,48 +1,52 @@
+# frozen_string_literal: true
+
 require 'test_helper'
 
 class QuotesControllerTest < ActionDispatch::IntegrationTest
   setup do
     @quote = quotes(:one)
   end
 
-  test "should get index" do
+  test 'should get index' do
     get quotes_url
     assert_response :success
   end
 
-  test "should get new" do
+  test 'should get new' do
     get new_quote_url
     assert_response :success
   end
 
-  test "should create quote" do
+  test 'should create quote' do
     assert_difference('Quote.count') do
-      post quotes_url, params: { quote: { age: @quote.age, quote_cents: @quote.quote_cents, quote_currency: @quote.quote_currency, trip_length: @quote.trip_length } }
+      post quotes_url, params: { quote: {
+        age: @quote.age,
+        trip_length: @quote.trip_length
+      } }
     end
 
     assert_redirected_to quote_url(Quote.last)
   end
 
-  test "should show quote" do
-    get quote_url(@quote)
-    assert_response :success
+  test 'should not save the quote and flash an error message if the quote builder fails' do
+    mock_organizer_response = Minitest::Mock.new
```

Comment:
I deliberately stuck with Minitest because I wanted to see how it felt to use all the Rails defaults. This is the first Minitest I've written with mocks. It feels incredibly clumsy compared to RSpec's mocking. I already preferred RSpec's readability over Minitests. I won't choose Minitest again in future projects.